api/bazel/api_build_system.bzl

```diff
@@ -7,7 +7,7 @@ _PY_SUFFIX = "_py"
 _CC_SUFFIX = "_cc"
 _CC_EXPORT_SUFFIX = "_export_cc"
 _GO_PROTO_SUFFIX = "_go_proto"
-_GO_IMPORTPATH_PREFIX = "github.com/envoyproxy/data-plane-api/api/"
+_GO_IMPORTPATH_PREFIX = "github.com/envoyproxy/go-control-plane/"
```

Comment:
How does this impact local development flows?

Reply:
This aligns the generated/bazel-internal build with the public bazel-less build, so I think it's an improvement.